internal/pkg/manifest/transform.go

```diff
@@ -305,13 +313,23 @@ func (t unionTransformer[Basic, Advanced]) Transformer(typ reflect.Type) func(ds
 			var zero Advanced
 			dstStruct.isAdvanced, dstStruct.Advanced = false, zero
 			dstStruct.isBasic = true
+
+			switch any(srcStruct.Basic).(type) {
+			case string, bool, int, int8, int16, int32, int64, uint8, uint16, uint32, uint64, float32, float64, complex64, complex128:
```

Comment:
is this different than that is defined in `basicKinds`?

Reply:
removed! not needed after #4189